views/CurrenciesList/index.tsx

```diff
@@ -1,30 +1,36 @@
 import React, {useEffect} from 'react';
-import { useDispatch, useSelector } from 'react-redux';
 import { Text, TouchableOpacity, View, Image, FlatList, ListRenderItem} from 'react-native';
-import {getUserCurrenciesFromAsync} from '../../redux/actions/currenciesActions';
+import { useSelector, useDispatch } from 'react-redux';
 import { IState } from '../../redux/reducers/mainReducer';
 import CurrencyCard from '../../components/CurrencyCard';
-import { useNavigation } from '@react-navigation/native';
-import styles from './styles';
-import { ICurrency } from '../../types';
 import { Routes } from '../../navigation/routes';
+import { ICurrency } from '../../types';
+import styles from './styles';
+import { updateCurrencyList, getUserCurrenciesFromAsync } from '../../redux/actions/currenciesActions';
 
-const Home = (): JSX.Element => {
-  const dispatch = useDispatch();
-  const {navigate}: any = useNavigation();
+const Home = ({navigation: {navigate}}: any): JSX.Element => {
+  const userCurrencyList = useSelector( (state: IState)=> state.currencies.userCurrencyList)
   const {ADD_CURRENCY} = Routes
 
+  const dispatch = useDispatch()
+
   useEffect(() => {
     dispatch(getUserCurrenciesFromAsync())
+
+    const intervalID = setInterval(() => {
+      dispatch(updateCurrencyList())
+    }, 25000)
```

Comment:
[CRITICAL] You should validate that the user has added cryptos on their list before starting an interval